src/main/java/org/isf/config/SpringFoxConfig.java

```diff
@@ -50,55 +49,58 @@
-        protocols.add(protocol);
-
-
-        return new Docket(DocumentationType.SWAGGER_2)
-                .apiInfo(apiInfo)
-                .host(host)
-                .protocols(protocols)
-                .select()
-                .apis(RequestHandlerSelectors.basePackage("org.isf"))
-                //.apis(RequestHandlerSelectors.any())
-                .paths(PathSelectors.any())
-                .build()
-                .securityContexts(Arrays.asList(jwtSecurityContext()))
-                .securitySchemes(Arrays.asList(apiKey()));
-    }
-
-    private SecurityContext jwtSecurityContext() {
-        return SecurityContext
-                .builder()
-                .securityReferences(defaultAuth())
-                .forPaths(PathSelectors.regex("^(?!(\\/auth\\/login)).*$"))
-                .build();
-    }
-
-    List<SecurityReference> defaultAuth() {
-        return Arrays.asList(new SecurityReference("JWT", new AuthorizationScope[0]));
-    }
-
-
-    private SecurityReference apiKeyReference() {
-        return new SecurityReference("Authorization", new AuthorizationScope[0]);
-    }
-
-    private ApiKey apiKey() {
-        return new ApiKey("Authorization", HttpHeaders.AUTHORIZATION, In.HEADER.name());
-    }
+	@Autowired
+	private Environment env;
+
+	@Bean
+	public Docket apiDocket() {
+
+//        List<SecurityScheme> securitySchemes = Arrays.asList(new ApiKey("JWT", "Authorization", "header"));
+
+		ApiInfo apiInfo = new ApiInfo("OH 2.0 Api Documentation", "OH 2.0 Api Documentation", "1.0", "urn:tos", ApiInfo.DEFAULT_CONTACT, "Apache 2.0",
+						"https://www.apache.org/licenses/LICENSE-2.0", Collections.emptyList());
+
+		String host = env.getProperty("api.host");
+		String protocol = env.getProperty("api.protocol");
+
+		Set<String> protocols = new HashSet<>();
+		protocols.add(protocol);
+
+		return new Docket(DocumentationType.SWAGGER_2)
+						.apiInfo(apiInfo)
+						.host(host)
+						.protocols(protocols)
+						.select()
+						.apis(RequestHandlerSelectors.basePackage("org.isf"))
+						// .apis(RequestHandlerSelectors.any())
+						.paths(PathSelectors.any())
+						.build()
+						.securityContexts(Arrays.asList(jwtSecurityContext()))
+						.securitySchemes(Arrays.asList(apiKey()));
+	}
+
+	private SecurityContext jwtSecurityContext() {
+		return SecurityContext
+						.builder()
+						.securityReferences(defaultAuth())
+						// .forPaths(PathSelectors.regex("^(?!(\\/auth\\/login)).*$"))
+						.build();
+	}
+
+	List<SecurityReference> defaultAuth() {
+		// return Arrays.asList(new SecurityReference("JWT", new AuthorizationScope[0]));
+		AuthorizationScope authorizationScope = new AuthorizationScope("global", "accessEverything");
+		AuthorizationScope[] authorizationScopes = new AuthorizationScope[1];
+		authorizationScopes[0] = authorizationScope;
+		return Arrays.asList(new SecurityReference("JWT", authorizationScopes));
+	}
+
+//    private SecurityReference apiKeyReference() {
```

Comment:
why keep any old, no longer used commented out code?   It can always be found in the git history can't it?

Reply:
yes, done, thanks!